torchaudio/csrc/sox/utils.cpp

```diff
@@ -417,8 +424,12 @@ unsigned get_precision(const std::string filetype, caffe2::TypeMeta dtype) {
   if (filetype == "amr-nb") {
     return 16;
   }
-  if (filetype == "gsm")
+  if (filetype == "htk") {
     return 16;
+  }
+  if (filetype == "gsm") {
+    return 16;
+  }
   throw std::runtime_error("Unsupported file type: " + filetype);
```

Comment:
While this is technically correct, can you add HTK after GSM so that diff is simpler?

Reply:
Done. Thanks!
All builds are failing for unrelated reasons, BTW.